### ISSUE TITLE
Using custom client ERROR

### ISSUE THREAD

**Issue by RelTakeover** [OgarProject/Ogar#668]
when i used custom client throw me this error :+1: 

Uncaught RangeError: Offset is outside the bounds of the DataView(…)

how to fix?

**Comment by NatsuTheGreat** [comment]
Obviously you modified something?

**Comment by RelTakeover** [comment]
i use the main_out of Ogarplus.js desencrypted

**Comment by RelTakeover** [comment]
the names change to chinese letters and have lagg with this client and the error said "Uncaught RangeError: Offset is outside the bounds of the DataView(…)"

picture: CONSOLE ERROR

![image](https://cloud.githubusercontent.com/assets/19374681/19774906/074f0e36-9c45-11e6-9938-e5742b90389e.png)


PICTURE GAME CUSTOM CLIENT WITH OGARPROJECt:

![image](https://cloud.githubusercontent.com/assets/19374681/19774927/184b1b6c-9c45-11e6-800a-740f5d914078.png)

error CONSOLE GOOGLE :+1: 

![image](https://cloud.githubusercontent.com/assets/19374681/19774958/2c0ec7a2-9c45-11e6-9574-6ae00464fbe8.png)

**Comment by RelTakeover** [comment]
FROM THE CLIENT OF ALIS.IO SAME ERROR

![image](https://cloud.githubusercontent.com/assets/19374681/19775047/7a2721c8-9c45-11e6-9015-b6e70a9d24d5.png)

**Comment by NatsuTheGreat** [comment]
You're using OgarPlus?

**Comment by RelTakeover** [comment]
yes how to fix this errors?

**Comment by NatsuTheGreat** [comment]
Well no wonder, that's so outdated. Use something up-to-date.

**Comment by ZfsrGhS953** [comment]
The problem seems to be this line: https://github.com/OgarProject/Ogar/blob/master/src/packet/UpdateNodes.js#L81 It should be 
```javascript
            buffer.setUint16(0);                                                 // Node name terminator
```

**Comment by Luka967** [comment]
Yep I keep forgetting to change that one, @ZfsrGhS953 is correct

**Comment by ItzLevvie** [comment]
@RelTakeover For the console error

Use this https://github.com/Luka967/Ogar/blob/d6464169f056413e9298f7d1de1783abe1b49174/src/gamemodes/FFA.js#L58
